packages/git/writer.js

```diff
@@ -71,7 +95,15 @@ module.exports = class Writer {
 
     await this._ensureRepo();
     return withErrorHandling(id, type, async () => {
-      let change = await Change.create(this.repo, document.meta.version, this.branchPrefix + branch);
+      const branchName = this.branchPrefix + branch;
+      let change;
+
+      if(this.remote) {
+        change = await Change.createRemote(this.repo, branchName, this.fetchOpts);
```

Comment:
I don't think this can be correct. It's ignoring `document.meta.version`, which is really important.

In general, I think the distinction between `create` and `createRemote` is probably not needed / actively harmful.